### ISSUE TITLE
"Process could not be started" during docker build

### ISSUE THREAD

**Issue by mfojtik** [docker/docker#10525]
I can see following error in Docker logs that appears every time I execute 'docker build -t IMAGE_NAME .':

UPDATE: Same error when doing 'docker run'...

```
Feb 03 09:44:22 naboo docker[1649]: time="2015-02-03T09:44:22+01:00" level="info" msg="POST /v1.16/build?rm=1&t=openshift%2Fsti-image-builder"
Feb 03 09:44:22 naboo docker[1649]: time="2015-02-03T09:44:22+01:00" level="info" msg="+job build()"
Feb 03 09:44:23 naboo docker[1649]: time="2015-02-03T09:44:23+01:00" level="info" msg="+job allocate_interface(05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5)"
Feb 03 09:44:23 naboo docker[1649]: time="2015-02-03T09:44:23+01:00" level="info" msg="-job allocate_interface(05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5) = OK (0)"
Feb 03 09:44:23 naboo docker[1649]: time="2015-02-03T09:44:23+01:00" level="info" msg="+job log(start, 05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5, centos:centos7)"
Feb 03 09:44:23 naboo docker[1649]: time="2015-02-03T09:44:23+01:00" level="info" msg="-job log(start, 05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5, centos:centos7) = OK (0)"
Feb 03 09:44:28 naboo docker[1649]: time="2015-02-03T09:44:28+01:00" level="info" msg="+job release_interface(05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5)"
Feb 03 09:44:28 naboo docker[1649]: time="2015-02-03T09:44:28+01:00" level="info" msg="-job release_interface(05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5) = OK (0)"
Feb 03 09:44:28 naboo docker[1649]: time="2015-02-03T09:44:28+01:00" level="info" msg="+job release_interface(05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5)"
Feb 03 09:44:28 naboo docker[1649]: time="2015-02-03T09:44:28+01:00" level="info" msg="-job release_interface(05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5) = OK (0)"
Feb 03 09:44:28 naboo docker[1649]: time="2015-02-03T09:44:28+01:00" level="error" msg="Warning: error unmounting device 05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5: UnmountDevice: device not-mounted id 05f15f34998f4d01973bf4097bc1b6a6479294d75daac7ef8737a02d4ae086b5\n"
Feb 03 09:44:28 naboo docker[1649]: Process could not be started
Feb 03 09:44:28 naboo docker[1649]: time="2015-02-03T09:44:28+01:00" level="info" msg="-job build() = ERR (1)"
```

The `docker info` output:

```
Containers: 3
Images: 71
Storage Driver: devicemapper
 Pool Name: docker-8:3-17071968-pool
 Pool Blocksize: 65.54 kB
 Data file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata file: /var/lib/docker/devicemapper/devicemapper/metadata
 Data Space Used: 4.027 GB
 Data Space Total: 107.4 GB
 Metadata Space Used: 4.895 MB
 Metadata Space Total: 2.147 GB
 Library Version: 1.02.92 (2014-11-28)
Execution Driver: lxc-1.1.0
Kernel Version: 3.18.5-1-ARCH
Operating System: Arch Linux
CPUs: 4
Total Memory: 7.532 GiB
Name: naboo
ID: NZI2:LBG7:K7IA:7U4J:2P3I:Q5XS:IJX6:QFHE:Z6OQ:JFAM:BVFI:HLAF
Username: mfojtik
Registry: [https://index.docker.io/v1/]
WARNING: No swap limit support
```

**Comment by mfojtik** [comment]
This is reproducible with the latest master as well. Removing /var/lib/docker/* does not help either.

**Comment by vbatts** [comment]
see https://gist.github.com/mfojtik/065ce07a6398fde8b242 for additional info

**Comment by vbatts** [comment]
also, this appears to be the same as #4036 but on 01b46c2 and udev sync shows to be true :-(

**Comment by vbatts** [comment]
conversation https://botbot.me/freenode/docker-dev/msg/31111573/

**Comment by mfojtik** [comment]
Two possible workarounds here:

1) Use 'native' driver which seems to work
2) Downgrade to lxc < 1.1.0 which seems to fix this problem as well

**Comment by ashahab-altiscale** [comment]
@mfojtik , at this point lxc-1.0.7 is the supported version. Feel free to submit patches to make docker more compatible with lxc-1.1.0. Thanks for posting the issue.

**Comment by calavera** [comment]
Closed, since it was fixed with #10527
